containers/gems/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN set -ex && \
 # install python, pip,  and pytest \
     apk add --no-cache python3=3.8.5-r0 && \
     python3 -m ensurepip && \
-    pip3 install pytest==6.1.1 && \
+    pip3 install --no-cache-dir pytest==6.1.1 && \
```

Comment:
good catch, perhaps this is a copy/pasta side-effect because this is supposed to be the gems container, and not really one where Python packages should be installed. Good change regardless!

Reply:
interesting, hadn't considered that it wasn't needed 🤔 . I think I'll leave this for now as this gets linting to pass, but you're right it seems that we could remove a few things from this image.